Localization/en-us.cfg

```diff
@@ -0,0 +1,183 @@
+        #Principia_OBTAnalyser_Resault_GroundCycles =  (<<1>> ground track cycles) //<<1>> ground_track_cycles.FormatN(0)
+        #Principia_OBTAnalyser_Resault_ListRevolutions = <<1>> sidereal revolutions\n<<2>> nodal revolutions<<3>>\n<<4>> anomalistic revolutions //<<3>>duration_in_ground_track_cycles
+        #Principia_OBTAnalyser_Resault_DurationgRecolutions = could not determine elements; mission duration may be shorter than a revolution, or trajectory may not be gravitationally bound
+        #Principia_OBTAnalyser_Resault_DurationgRecolutions2 =  to <<1>>. //<<1>> primary.NameWithArticle()
+        #Principia_OBTAnalyser_Resault_AnalysisDescript = <<1>> is not gravitationally bound over <<2>> //<<2>>mission_duration.FormatDuration <<1>> vessel name
+        #Principia_OBTAnalyser_Resault_AnalysisDescript2 = Orbit of <<1>> with respect to <<2>> over <<3>>:\n<<4>> //<<1>> vesselname <<2>> centre body <<3>> time <<4>> resault
+        #Principia_OBTAnalyser_OBTCategory_Circular = circular 
+        #Principia_OBTAnalyser_OBTCategory_High = highly elliptical 
+        #Principia_OBTAnalyser_OBTCategory_Equat = equatorial 
+        #Principia_OBTAnalyser_OBTCategory_Polar = polar 
+        #Principia_OBTAnalyser_OBTCategory_Retrograde = retrograde
+        #Principia_OBTAnalyser_OBTCategory_Synchronous = synchronous 
+        #Principia_OBTAnalyser_OBTCategory_Stationary = stationary
+        #Principia_OBTAnalyser_OBTCategory_Semisynch = semisynch.  
+        #Principia_OBTAnalyser_OBTCategory_Subsynch = subsynch.
+        #Principia_OBTAnalyser_OBTCategory_Supersynch = supersynch. 
+        #Principia_OBTAnalyser_Resault_OBTCategory = <<1>> <<2>> orbit //<<1>>properties <<2>>primary.name
+        #Principia_OBTAnalyser_Resault_LowestAlt = Lowest altitude
+        #Principia_OBTAnalyser_Resault_AltWarn1 = collision
+        #Principia_OBTAnalyser_Resault_AltWarn2 = collision risk
+        #Principia_OBTAnalyser_Resault_AltWarn3 = reentry
+        #Principia_OBTAnalyser_Resault_OBTElements = Mean orbital elements
+        #Principia_OBTAnalyser_Resault_SiderealPeriod = Sidereal period
+        #Principia_OBTAnalyser_Resault_NodalPeriod = Nodal period
+        #Principia_OBTAnalyser_Resault_AnomalisticPeriod = Anomalistic period
+        #Principia_OBTAnalyser_Resault_SemimajorAxis = Semimajor axis
+        #Principia_OBTAnalyser_Resault_Eccentricity = Eccentricity
+        #Principia_OBTAnalyser_Resault_Inclination = Inclination
+        #Principia_OBTAnalyser_Resault_LAN = Longitude of ascending node
+        #Principia_OBTAnalyser_Resault_NodalPrecession = Nodal precession
+        #Principia_OBTAnalyser_Resault_ArgumentPE = Argument of periapsis
+        #Principia_OBTAnalyser_Resault_ALTPE = Altitude of mean periapsis
+        #Principia_OBTAnalyser_Resault_ALTAP = Altitude of mean apoapsis
+        #Principia_OBTAnalyser_Plugin_Recurrence = Recurrence: <<1>>; <<2>>; <<3>> //<<1>> nuo <<2>> dto <<3>> cto
```

Comment:
Keep the square brackets here, including in the Chinese version. [ν<sub>o</sub>; D<sub>T<sub>o</sub></sub>; C<sub>T<sub>o</sub></sub>] is a notation from [[Cap12]](https://github.com/mockingbirdnest/Principia/blob/b9981c096b5539fc0025a059c3230821a45d7880/documentation/bibliography.pdf), with a specific technical meaning; this is not just three numbers separated by semicolons.